### ISSUE TITLE
​Fresh install with NPM and React 18.1 fails

### ISSUE THREAD

**Issue by jamesgpearce** [tinyplex/tinybase#35]
### Describe the bug

Via @founderYonz

​Fresh install with NPM and React 18.1 fails because npm doesn't respect the package.json "optional: true" for peer Dependency.

```
    tinyBaseExpoNB $ npm i tinybase
    npm ERR! code ERESOLVE
    npm ERR! ERESOLVE could not resolve
    npm ERR! 
    npm ERR! While resolving: tinybase@2.1.1
    npm ERR! Found: react@18.1.0
```

**Comment by jamesgpearce** [comment]
Fixed in v2.2.4 - please verify! @yfeleke

**Comment by yfeleke** [comment]
Verified with 2.2.6
